includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -41,7 +41,37 @@ public function __construct() {
 		$this->total_label     = ! empty( $this->stripe_settings['statement_descriptor'] ) ? WC_Stripe_Helper::clean_statement_descriptor( $this->stripe_settings['statement_descriptor'] ) : '';
 
 		$this->total_label = str_replace( "'", '', $this->total_label ) . apply_filters( 'wc_stripe_payment_request_total_label_suffix', ' (via WooCommerce)' );
+	}
 
+	/**
+	 * Returns a list of the payment method titles.
+	 *
+	 * @param bool $include_link Whether to include the Link payment method.
+	 * @return array
+	 */
+	public static function get_payment_method_titles( $include_link = false ) {
+		$titles = [
+			'apple_pay'                           => WC_Stripe_Express_Payment_Titles::APPLE_PAY,
```

Comment:
Another implementation option would be translated names, like:
```php
'apple_pay' => __( 'Apple Pay', 'woocommerce-gateway-stripe' ),
```
But not sure if it is worth since those are brands, which I think are immutable.